CMakeLists.txt

```diff
@@ -16,11 +16,11 @@ SET_SOURCE_FILES_PROPERTIES(${PROTO_HEADERS} ${PROTO_SOURCES} PROPERTIES GENERAT
 # Generate python module
 add_subdirectory(lib/pybind11)
 pybind11_add_module(pybmixcpp ${SOURCES}
-    "${SOURCE_DIR}/module.cpp"
-    "${SOURCE_DIR}/algorithm_wrapper.hpp"
-    "${SOURCE_DIR}/algorithm_wrapper.cpp"
-    "${SOURCE_DIR}/serialized_collector.hpp"
-    "${SOURCE_DIR}/serialized_collector.cpp"
+    "${SOURCE_DIR}/bayesmix/module.cpp"
```

Comment:
The folders docs, example and test are there because if we remove them the make command complains about that, we will look for a way to deal with this. The internal bayesmix is the submodule, the external one is the folder in which we build the python module (we thought this would be OK since pybmix has a sub-folder named pybmix too). We reached this structure because we were trying to compile our files alongside the bayesmix submodule without changing it and this was the only solution we were able to find while writing the cmakefile.

Reply:
Ok, that should be telling you that your build system is broken...

About the naming: I don't think you need two bayesmix folders: in case of Python, this is needed because of the way python installs packages, and it's really ugly and hopefully will be fixed at some point.
Previously, the pybmixcpp folder was doing exactly the same thing as the external bayesmix folder is doing now. I feel it would be cleaner to have only of the two. You can change the pybmixcpp name if you don't like it of course, I'm open for suggestions :)

Reply:
Hi, 
we changed the build system:

- we removed the useless folders (e.g. examples, docs, ...) in pybmixcpp
- we restored bayesmix to the original location

Let us know if you like it :)